index.js

```diff
@@ -18,5 +18,6 @@ module.exports = {
     'func-names': 'off',
     'react/jsx-boolean-value': 'off',
     'no-multiple-empty-lines': ["error", { "max": 1 }],
+    'no-return-assign': ['error', 'except-parens'],
```

Comment:
Outta curiosity: What was the use-case for this? I didn't find an example in the corresponding bigmaven pull request.

Reply:
There's a handful of places (maybe 5 or 6 or so) in BigMaven that do a return and assigns in one go.

This rule is present already in the airbnb config, and here we configure it to allow this pattern when parens are around the assignment, which was the behavior before the upgrade, I think.

Probably makes sense to go back and remove those from bigmaven, though.

Reply:
UPDATE:

Heh, when removing this rule I actually only found 1 example in big maven. Would've sworn there were more...